src/Sql/Sql/Common/Constants.cs

```diff
@@ -32,5 +32,7 @@ public class Constants
         public const string ComputeGenerationGen5 = "Gen5";
         public const string CollationSqlLatin1 = "SQL_Latin1_General_CP1_CI_AS";
         public const string CollationLatin1 = "Latin1_General_100_CS_AS_SC";
+        public const string PublicDataEndpointEnabledTrue = "$true";
```

Comment:
These should not be needed. PowerShell should already know what values are allowed for boolean type.